build.gradle

```diff
@@ -37,45 +37,198 @@ allprojects {
-        maven { url 'https://gitlab.matrix.org/api/v4/projects/27/packages/maven' }
-
+        // For olm library.
+        maven {
+            url 'https://gitlab.matrix.org/api/v4/projects/27/packages/maven'
+            content {
+                includeGroup 'org.matrix.android'
+            }
+        }
         maven {
             url 'https://jitpack.io'
             content {
-                // Use this repo only for FilePicker
-                includeGroupByRegex "com\\.github\\.jaiselrahman"
-                // And monarchy
-                includeGroupByRegex "com\\.github\\.Zhuinden"
-                // And ucrop
-                includeGroupByRegex "com\\.github\\.yalantis"
-                // JsonViewer
-                includeGroupByRegex 'com\\.github\\.BillCarsonFr'
-                // PhotoView
-                includeGroupByRegex 'com\\.github\\.chrisbanes'
-                // PFLockScreen-Android
-                includeGroupByRegex 'com\\.github\\.vector-im'
-                // DraggableView
-                includeGroupByRegex 'com\\.github\\.hyuwah'
-
-                // Chat effects
-                includeGroupByRegex 'com\\.github\\.jetradarmobile'
-                includeGroupByRegex 'nl\\.dionsegijn'
-
-                // Voice RecordView
-                includeGroupByRegex 'com\\.github\\.Armen101'
+                includeGroup 'com.github.Armen101'
+                includeGroup 'com.github.BillCarsonFr'
+                includeGroup 'com.github.chrisbanes'
+                includeGroup 'com.github.hyuwah'
+                includeGroup 'com.github.jetradarmobile'
+                includeGroup 'com.github.tapadoo'
+                includeGroup 'com.github.vector-im'
+                includeGroup 'com.github.yalantis'
+                includeGroup 'com.github.Zhuinden'
             }
         }
-        maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
         // Jitsi repo
         maven {
             url "https://github.com/vector-im/jitsi_libre_maven/raw/main/android-sdk-3.10.0"
             // Note: to test Jitsi release you can use a local file like this:
             // url "file:///Users/bmarty/workspaces/jitsi_libre_maven/android-sdk-3.10.0"
+            content {
+                includeGroup 'com.facebook.react'
+                includeGroup 'org.jitsi.react'
+                includeGroup 'org.webkit'
+            }
+        }
+        google {
+            content {
+                includeGroupByRegex "androidx\\..*"
+                includeGroupByRegex 'com\\.android\\.tools\\..*'
+                includeGroupByRegex "com\\.google\\.android\\..*"
+                includeGroup 'com.google.firebase'
+                includeGroup 'com.android'
+                includeGroup 'com.android.tools'
+            }
+        }
+        mavenCentral {
+            content {
+                includeGroup 'com.adevinta.android'
```

Comment:
if only we had a way to externalize those list into another file...
`includeGroup` does not support taking a list as param

Reply:
let me check if that is possible